packages/web-react/src/components/FieldGroup/__tests__/FieldGroup.test.tsx

```diff
@@ -54,7 +58,7 @@ describe('FieldGroup', () => {
 
   it('should have className isRequired', () => {
     const dom = render(
-      <FieldGroup label="Label" isRequired>
+      <FieldGroup id="exampleFieldGroup" label="Label" isRequired>
```

Comment:
We have a rule (that is not written anywhere 🙂) that IDs should use kebab-case notation. Will you please change it? 🙂